src/components/layouts/Page/Page.styles.js

```diff
@@ -1,18 +1,41 @@
-import { css } from '@emotion/core';
+import { css } from 'lib/utils/media-queries';
+import { Color, Space } from 'lib/theme';
 
-import { Space } from 'lib/theme';
+const paddingDesktop = 22;
+const additionalRightPadding = paddingDesktop * 0.2;
+const rightPaddingDesktop = paddingDesktop + additionalRightPadding;
```

Comment:
Trying to use percentages here so the page keeps the design proportions, but it feels a little messy. The left panel has additional right padding that the other side doesn't have. Would welcome any suggestions here!

Reply:
hm... i wonder if that's intentional or not. i might check with virginie

Reply:
Let's check with design to see what the expectation is. A static padding may be preferred to maintain a grid (if we even have a grid 😄)

Reply:
Liz beat me to it!

Reply:
I don't think we have a grid yet. I'm betting on it being intentional because it looks better that way 😀 but I will check with Virginie to be sure.

Reply:
@lizbanach @joshrounsville I hope you'll forgive me 🙏but I'm merging as is for now until I get an answer on this. I will add an issue for myself to double check and fix if needed. Thanks!